### PR TITLE
tests: add logging integration tests

### PR DESCRIPTION
**Pull request by misiekn**
Include Monolog2 and Monolog3 bundles to be used in logging integration
tests.

Test expectations - metrics and log events - will be added in a separate PR.

### CODE REVIEW

File: tests/integration/logging/monolog2/test_monolog.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<?php
+
+require('skipif.inc');
+
+*/
+
+/*EXPECT
+monolog2.DEBUG: debug []
+monolog2.INFO: info []
+monolog2.NOTICE: info []
```
Comment (mfulb) [inline tests/integration/logging/monolog2/test_monolog.php:21]:
```suggestion
monolog2.NOTICE: notice []
```

File: tests/integration/logging/monolog3/test_monolog.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<?php
+
+require('skipif.inc');
+
+*/
+
+/*EXPECT
+monolog3.DEBUG: debug []
+monolog3.INFO: info []
+monolog3.NOTICE: info []
```
Comment (mfulb) [inline tests/integration/logging/monolog3/test_monolog.php:21]: ### 
```suggestion
monolog3.NOTICE: notice []
```

File: tests/integration/logging/monolog3/test_monolog.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    $formatter = new LineFormatter($logfmt);
+
+    $stdoutHandler = new StreamHandler('php://stdout', Logger::DEBUG);
+    $stdoutHandler->setFormatter($formatter);
+
+    $logger->pushHandler($stdoutHandler);
+    
+    $logger->debug("debug");
+    $logger->info("info");
+    $logger->notice("info");
```
Comment (mfulb) [inline tests/integration/logging/monolog3/test_monolog.php:51]:
```suggestion
    $logger->notice("notice");
```

File: tests/integration/logging/monolog2/test_monolog.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    $formatter = new LineFormatter($logfmt);
+
+    $stdoutHandler = new StreamHandler('php://stdout', Logger::DEBUG);
+    $stdoutHandler->setFormatter($formatter);
+
+    $logger->pushHandler($stdoutHandler);
+    
+    $logger->debug("debug");
+    $logger->info("info");
+    $logger->notice("info");
```
Comment (mfulb) [inline tests/integration/logging/monolog2/test_monolog.php:51]:
```suggestion
    $logger->notice("notice");
```

**Comment by misiekn** [comment]
ok jenkins

**Comment by misiekn** [comment]
ok jenkins

**Review by mfulb — APPROVED** [review]
Looks great Michal thanks for getting this integrated.

File: tests/include/monolog.php [hunk 5]
```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
```
Comment (zsistla) [inline tests/include/monolog.php:3]: 2022?

**Review by zsistla — APPROVED** [review]
Looks good, thanks for expanding our tests!

File: tests/include/monolog.php (hunk 5 above)
Comment (misiekn) [reply tests/include/monolog.php]: Addressed in [146d6ec](https://github.com/newrelic/newrelic-php-agent/pull/502/commits/146d6eccd9807e27350f6cb1bb1e53b2661f5054)

**Comment by misiekn** [comment]
ok jenkins

**Pull request merged**